web/index.html

```diff
@@ -48,7 +48,11 @@
     <textarea id="input" rows="10" cols="80" disabled>Input</textarea>
     <textarea id="output" rows="10" cols="80" disabled>Output</textarea>
 
-    <script>
+    <script type="module">
+      import init, { js_tags_to_lanes } from "./pkg/osm2lanes.js";
+
+      await init();
```

Comment:
I'm not convinced this is correct; I think everything here needs to be in an `async function`. But it kind of works out as long as the user doesn't click a road too quickly before this happens to finish

Reply:
I think [top level await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#top_level_await) is a thing (though I personally haven't used module scripts directly in the browser before). So this should work, but at the cost of queuing the rest of the setup work behind `init()`.

You could kick off `init()` here, store the promise, and `await` it just before you use the lib (on line 120).

Reply:
Hmm, then we need extra state to avoid awaiting the promise multiple times. Cool that top-level await works directly though! I'll keep this in mind if we hit issues later

Reply:
FYI, awaiting the promise multiple times is also safe, at the cost of yielding to the event loop (like `setTimeout( _, 0)`).